src/Jackett.Common/Definitions/hdcity.yml

```diff
@@ -179,7 +179,7 @@
         optional: true
         filters:
           - name: append
-            args: " [Spanish]"
+            args: " [Spanish] [English]"
```

Comment:
This clashes with #6338, where [spanish] [english] was deliberately changed to [Spanish] for non-VOSE tags.
@rubasace  please talk with @TuGeek and agree on which you want to have.
If you cannot agree then a 3rd option is to set up a config box where you can load what ever you want in the non-VOSE tags.
Thanks.

Reply:
@ngosang I'm afraid I don't have any, as registration is closed. 

Nice catch. I'd like to know @TuGeek opinion on the matter, cannot think of a reason why English is a problem there, but sure I'm missing something. A configuration box sounds like a good alternative otherwise, but let's see.

Reply:
I don't remember exactly what the reason was, but I think it was to clearly separate the VOSE content from the Spanish content.
I agree to change it to return the releases that are in DUAL.

Thanks @rubasace